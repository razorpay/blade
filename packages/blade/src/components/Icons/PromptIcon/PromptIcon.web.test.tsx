import PromptIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<PromptIcon />', () => {
  it('should render PromptIcon', () => {
    const { container } = renderWithTheme(
      <PromptIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
