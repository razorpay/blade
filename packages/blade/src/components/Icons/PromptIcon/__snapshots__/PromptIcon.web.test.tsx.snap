// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PromptIcon /> should render PromptIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      d="M2 6C2 5.44772 2.44772 5 3 5H21C21.5523 5 22 5.44772 22 6C22 6.55228 21.5523 7 21 7H3C2.44772 7 2 6.55228 2 6Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M2 12C2 11.4477 2.44772 11 3 11H21C21.5523 11 22 11.4477 22 12C22 12.5523 21.5523 13 21 13H3C2.44772 13 2 12.5523 2 12Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M3 17C2.44772 17 2 17.4477 2 18C2 18.5523 2.44772 19 3 19H11C11.5523 19 12 18.5523 12 18C12 17.4477 11.5523 17 11 17H3Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
  </svg>
</div>
`;
